_archives/refactor/uno/_quickstart.md

```diff
@@ -101,15 +101,15 @@ To communicate via The Things Network, you need to register your device. For thi
     Soon, it should print a list of information, including your **DevEUI**:
 
     ```
-    -- STATUS
-    EUI: 0004A30B001B7AD2
-    Battery: 3223
-    AppEUI: 70B3D57EF000001C
-    DevEUI: 0004A30B001B7AD2
-    DevAddr: 26012E93
-    Data Rate: 5
-    RX Delay 1: 1000
-    RX Delay 2: 2000
+    -- STATUS
```

Comment:
No idea what the change is here. Still 4 spaces, still somehow GH thinks it has changed.